src/System.Globalization.Extensions/src/System.Globalization.Extensions.csproj

```diff
@@ -19,17 +20,14 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'OSX_Release|AnyCPU' " />
```

Comment:
Can you remove the spaces at the start and end of these conditions for consistency